src/components/ToDos.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from "react";
+import ToDosFooter from "./toDos--footer.jsx";
+import ToDosList from "./toDos--list.jsx";
+let randomstring = require("randomstring");
+
+let localData = [];
+
+export default class ToDos extends Component {
+  state = {
+    data: [],
+    inputValue: ""
+  };
+
+  async componentDidMount() {
+    localData = JSON.parse(window.localStorage.getItem("localData")) || [];
+
+    this.setState(prevState => ({
+      data: [...prevState.data, ...localData]
```

Comment:
```suggestion
      data: localData
```
You don't have any useful `prevState` in `componentDidMount` hook